build.ps1

```diff
@@ -156,24 +158,30 @@ if (!(Test-Path $NUGET_EXE)) {
 }
 
 # Save nuget.exe path to environment to be available to child processed
-$ENV:NUGET_EXE = $NUGET_EXE
+$env:NUGET_EXE = $NUGET_EXE
+$env:NUGET_EXE_INVOCATION = if ($IsLinux -or $IsMacOS) {
```

Comment:
I like this. will have to use it in the other repos.

Reply:
hmm dont remember adding it